it/css/README.md

```diff
@@ -10,18 +10,18 @@ Ma non vogliamo ricominciare da capo, giusto? Ancora una volta useremo qualcosa
 
 ## Usiamo Bootstrap!
 
-Bootstrap è uno dei più popolari framework HTML e CSS per costruire bellissimi siti internet: https://getbootstrap.com/
+Bootstrap è uno dei più popolari framework HTML e CSS per costruire bellissimi siti internet: http://getbootstrap.com/
 
 È stato scritto da programmatori che lavoravano per Twitter ed è ora sviluppato da volontari da ogni parte del mondo.
 
 ## Installa Bootstrap
 
 Per installare Bootstrap, avrai bisogno di aggiungere questo nel tag `<head>` del tuo file `.html` (`blog/templates/blog/post_list.html`):
 
-```html
-<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
-<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
-```
+    html
```

Comment:
The fencing should definitelly stay. It's a more desired syntax of code highlighting.